packages/opentelemetry/lib/tracing/tracerProvider.js

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable no-unused-vars */
```

Comment:
why do we need this annotation?

Reply:
importing for types, linter doesnt recognize and throws error

Reply:
In VSCode we could use something like https://github.com/jsdoc/jsdoc/issues/1537#issuecomment-457231965

Reply:
also https://github.com/jsdoc/jsdoc/issues/1537#issuecomment-489614118